technical-doc/storage-costs.rst

```diff
@@ -29,5 +29,13 @@ To create your address on the blockchain someone need to send at least 0.00025 *
 
 You want to store your birth date that is 30 bytes long in your datastore so you need to send an operation that will create a key in your datastore using a SC. This operation will cost you in storage costs at least 0.00025 * (10 + 30) = 0.01 Massa.
 
-Now you want to delete this entry on your datastore then you will do it with a SC and you will be refunded of the storage costs (0.02 Massa).
+Now you want to delete this entry on your datastore you will be refunded of the storage costs (0.02 Massa).
 
+In case you are interacting with a SC that will store the data on his datastore then the SC will pay for the storage costs not you as a TX sender (but he can use the coins you pass to the call to pay).
```

Comment:
I think the english can be (greatly) improved here.

Reply:
I have made a new formulation.